exercises/00-init/src/index.tsx

```diff
@@ -0,0 +1,8 @@
+import 'babel-polyfill'
```

Comment:
Might not be necessary anymore. Anyway, it's deprecated https://babeljs.io/docs/en/babel-polyfill.

Reply:
I've removed the import from this file.

The package will be removed from package.json when all exercises are updated. I've created an issue for that, see https://github.com/salsita/react-training/issues/39